build-system/tasks/process-github-issues.js

```diff
@@ -119,67 +124,118 @@ function updateGitHubIssues() {
           let milestoneTitle;
           let milestoneState;
           let hasPriority = false;
+          let hasCategory = false;
           let issueNewMilestone = MILESTONE_PENDING_TRIAGE;
-
-          // Get the title and state of the milestone
+          const assignee = issue['assignee'];
+          let assigneeName = '';
+          const issueLastUpdate = issue.updated_at;
+          let biweeklyUpdate = true;
+          let quartelyUpdate = true;
+          if (getLastUpdate(issueLastUpdate) > 89) {
+            quartelyUpdate = false;
+            biweeklyUpdate = false;
+          }
+          else if (getLastUpdate(issueLastUpdate) > 14) {
+            biweeklyUpdate = false;
+          }
+      // Get the assignee
```

Comment:
All the comments in this file are badly indented. This is because the linter has no way of knowing if they belong to the block before it or the block after it, and therefore, cannot fix them. You're going to have to manually fix them.

Reply:
done